test/BUILD.gn

```diff
@@ -233,6 +233,7 @@ test("brave_unit_tests") {
 
   if (ipfs_enabled) {
     sources += [
+      "//brave/browser/ipfs/content_browser_client_helper_unittest.cc",
```

Comment:
I'll refactor all this test gn code outside of test/BUILD.gn into its own target in a separate PR.